### ISSUE TITLE
COE launches automatically before I want it to

### ISSUE THREAD

**Issue by avh-agrointelli** [into-cps/intocps-ui#248]
_For INTO-CPS App latest version (v3.1.2-rc) for Windows x64_

I'm try to configure a DSE in the INTO-CPS App, and the COE launches automatically before I want it to, even after restarting the App. Because of this (I think) the multi-model cannot be saved and consequently the DSE config cannot be loaded and parsed.
Is it something I'm doing wrong or a bug?

**Comment by CarlGamble** [comment]
Yes, there is a bug where the DSE screens don't let you save your multi-model configuration choice if the COE is already running.  Until that bug is fixed, the workaround is to shut down the coe before trying to configure and launch a DSE.  This may be achieved in two ways.
1.  There is a blue gear like icon in the windows system tray.  Right clicking on this should bring up an option to shut down the coe.
2.  If option 1 fails to shut down the COE then the COE may be shut down using the windows task manager.  crtl-alt-delete and then select task manager.  The COE is a java process so you will need to find the instance of Java in which the coe is running and select end task.  The COE process is usually quite large.

**Comment by avh-agrointelli** [comment]
Thank you. Option 1 didn't work, but 2 did :)
